protobuf/flow/access/access.proto

```diff
@@ -409,3 +448,120 @@ message ExecutionResultByIDResponse {
   entities.ExecutionResult execution_result = 1;
   entities.Metadata metadata = 2;
 }
+
+// Subscriptions
+
+// The request for SubscribeBlocks
+message SubscribeBlocksRequest {
```

Comment:
Since we are introducing new API I suggest to enforce "having only `start_block_id` or `start_block_height`" in protobuf using [oneof](https://protobuf.dev/programming-guides/proto3/#oneof) keyword. This will make a cleaner API and simplify implementation.